rules/windows/evasion_suspicious_scrobj_load.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies scrobj.dll loaded into unusual Microsoft processes. This usually means a malicious scriptlet is being
+executed in the target process.
+"""
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Suspicious Script Object Execution"
+risk_score = 21
+rule_id = "4ed678a9-3a4f-41fb-9fea-f85a6e0a0dff"
+severity = "medium"
+tags = ["from-eql-rules"]
+type = "eql"
+
+query = '''
+sequence by process.entity_id, process.pid, process.name with maxspan=2m
+  [process where event.type in ("start", "process_started") and
+     process.code_signature.subject_name == "Microsoft Corporation" and process.code_signature.trusted == true and
+     not process.name in ("cscript.exe",
```

Comment:
This seems like a really small allowlist. I think we'll have an untenable amount of FPs.
Could we do the inverse and instead make a blacklist?